alvr/xtask/src/main.rs

```diff
@@ -38,6 +38,7 @@ FLAGS:
     --oculus-quest      Oculus Quest build. Used only for build-client subcommand
     --oculus-go         Oculus Go build. Used only for build-client subcommand
     --bundle-ffmpeg     Bundle ffmpeg libraries. Only used for build-server subcommand on Linux
+    --no-nvidia         Additional flag for build-ffmpeg-linux subcommand. Build don't reqired NVidia libs ffmpeg
```

Comment:
Is it possible to have a positive option (i.e. `--bundle-ffmpeg-nvenv` for instance)? Negative options always end up as a mess.

Reply:
I agree about the mess, but in general, it's only for developers who want to create a server with `--bundle-ffmpeg` but want to install a full pack of libs. I think is fine here, because usually, developers will build without the bundle ffmpeg inside and will link to existing libs to /usr/libs which means don't need to pass any flags at all.